apps/chat/src/components/Common/FolderContextMenu.tsx

```diff
@@ -124,6 +130,15 @@ export const FolderContextMenu = ({
         onClick: onPublish,
         disabled: disableAll,
       },
+      {
+        name: t('Unpublish'),
+        dataQa: 'unpublish',
+        display:
+          isPublishingEnabled && isItemPublic(folder.id) && !!onUnpublish,
+        Icon: UnpublishIcon,
+        onClick: onUnpublish,
+        disabled: disableAll,
+      },
```

Comment:
@Alexander-Kezik , what's about the code on line 154? 
Minimum it's duplicate.
Secondly line 154 is better place, because `Unpublish` should be after `Update`